tests/acceptance/3rdparty/mendertesting/check_commits.sh

```diff
@@ -20,11 +20,11 @@ fi
 if [ -n "$1" ]
 then
     echo "Checking range: $@:"
-    git log "$@"
+    git --no-pager log "$@"
     commits="$(git rev-list --no-merges "$@")"
 else
     echo "Checking range: ${COMMIT_RANGE}:"
-    git log "$COMMIT_RANGE"
+    git --no-pager log "$COMMIT_RANGE"
```

Comment:
This should be fixed upstream. Can you submit the same thing to https://github.com/mendersoftware/mendertesting ?